src/python_testing/TC_TIMESYNC_2_2.py

```diff
@@ -46,6 +46,9 @@ async def test_TC_TIMESYNC_2_2(self):
         endpoint = 0
 
         time_cluster = Clusters.Objects.TimeSynchronization
+        timesync_attr_list = Clusters.Objects.TimeSynchronization.Attributes.AttributeList
```

Comment:
changes as above

Reply:
Resolved using your above mentioned suggestions in TC_TIMESYNC_2_1 test module